jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1424,7 +1427,7 @@
     (let (avs)
       (setq avs
             (list
-              (send *baxter* :approach-from-downside arm)
+              (send *baxter* :midpose-hold-opposite-hand-object-downside arm)
```

Comment:
> i don't like that long name.

I think readability of the code is more important, because the hard coded pose is only used once and its longness does not effect much to whole code.

We can shorten the method name, for example renaming `:hold-opposite-hand-object` to `:hold-invhand-object` and using `:midpose-hold-invhand-object`.
Maybe `-object` is not necessary and we can use `:hold-invhand-downside` and `:midpose-hold-invhand-downside`.